src/main/java/com/quirko/logic/bricks/RandomBrickGenerator.java

```diff
@@ -21,6 +23,18 @@ public RandomBrickGenerator() {
         brickList.add(new SBrick());
         brickList.add(new TBrick());
         brickList.add(new ZBrick());
+        if(diff != DifficultyType.EASY){//extra hard shape for increase difficulty

```

Comment:
Try a different approach than limiting bricks types in each difficulty level.

Instead, play with the random generator's probability of choosing certain shapes for each level.

So all bricks should appear in all modes, but some modes may have more chance to have easier shapes.

Reply:
Hocam aslında bu yöntemde de tüm modeller her zorluk seviyesinde görünüyor ama zorsa daha çok zor şekil eklediğim için olasılığı bu şekilde arttırmış oldum. Generator'un seçtiği listeye zor seviyede daha çok zor şekil ekledim, kolay seviyede daha çok kolay şekil ekledim yani olasılığı değiştirmiş oldum böylece. Ama bunu kabul etmezseniz farklı bir şekilde olasılık değişimi yapmayı düşüneyim.